src/management/index.js

```diff
@@ -3894,4 +3902,24 @@ utils.wrapPropertyMethod(
   'prompts.updateCustomTextByLanguage'
 );
 
+/**
```

Comment:
You can remove these 2 aliases - going forward we're just going to use the `managment.namespace.*` ones

Reply:
thanks, I wasn't sure if I should have them